numpy/core/tests/test_numeric.py

```diff
@@ -140,6 +140,12 @@ def test_round(self):
         tgt = [1.6, 72.5, 6.4, 3.2]
         assert_equal(np.around(arr, decimals=1), tgt)
 
+        s = np.float64(1.)
+        assert_(isinstance(round(s), int))
+        assert_(isinstance(s.round(), np.generic))
```

Comment:
Let's be precise here
```suggestion
        assert_(isinstance(s.round(), np.float64))
```